### DISCUSSION TITLE
Total Row Results At Top

### DISCUSSION THREAD

**Discussion by HunterJacobs** [burkybang/SuiteAdvanced-Feedback#133] [Ideas]
When reviewing saved searches with a bunch of results I always have to scroll down to the bottom of the list to see the total row/results. It would be nice to have the ability for this appear at the top so we dont have to scroll down past 500 rows.

**Comment by burkybang** [comment]
I assume you're talking about a total that displays when summarizing. Is that right?

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/e950be80-af4b-45e5-9f7b-c7646128eb24)

**Reply by HunterJacobs** [reply]
That is correct

**Reply by burkybang** [reply]
I think that's a fantastic idea, and I've already completely a solution. The total row is now pinned to the bottom of the screen, so you can always see it no matter where you are scrolled. Thanks for the idea!

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/880e0f5d-94cc-4e31-86f1-858e10938a06)

**Reply by HunterJacobs** [reply]
Thank you for the extension and your ridiculously fast response/update!

**Reply by burkybang** [reply]
Haha no worries! This feature will be out in the next release.

**Reply by burkybang** [reply]
@Jacohun1 The update is out now!

https://suiteadvanced.com/version-29_3_0
